src/test/scala/mesosphere/marathon/api/akkahttp/PathMatchersTest.scala

```diff
@@ -66,6 +66,10 @@ class PathMatchersTest extends UnitTest with GroupCreation with ScalatestRouteTe
     "considers it an unmatch if path starts with keyword" in {
       AppPathIdLike(Path("/restart")) shouldBe Unmatched
     }
+
+    "match path with multiple slashes" in {
+      AppPathIdLike(Path("//////app")) shouldBe Matched(Path.Empty, Tuple1("//////app".toPath))
```

Comment:
I'm not sure if this is the behaviour we want to have. Let's ask @jeschkies

Reply:
well ... I tested it on the current API and it behaves like this... So I guess we must be backward compatible ... ?

Also what PathId does is that there is sanitization that actually removes all those //// and normalizes it to a correct PathId with just one /